optimum/exporters/onnx/__main__.py

```diff
@@ -61,6 +62,7 @@ def _get_submodels_and_onnx_configs(
     monolith: bool,
     custom_onnx_configs: Dict,
     custom_architecture: bool,
+    _variant: str,
```

Comment:
Why make it a protected parameter?

Reply:
I'm thinking to keep it "private" for now, and support it correctly once we move to this API fully instead of  `-with-past`, `monolith`, etc.